velox/substrait/VeloxToSubstraitExpr.h

```diff
@@ -96,6 +96,13 @@ class VeloxToSubstraitExprConvertor {
       google::protobuf::Arena& arena,
       const std::shared_ptr<ConstantVector<ComplexType>>& constantVector);
 
+  /// Convert Velox IN expression to Substrait SingularOrList.
+  const ::substrait::Expression& toSubstraitSingularOrList(
+      google::protobuf::Arena& arena,
+      const std::vector<std::shared_ptr<const core::ITypedExpr>>& inputs,
```

Comment:
maybe use TypedExprPtr to replace std::shared_ptr<const core::ITypedExpr>.

Reply:
Fixed cases like this. Thanks for your review.